montecarlo/sweep_mc_from_config.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import argparse
+from mcpy.monte_carlo import MonteCarloSweep
+import importlib
+
+def monte_carlo_main():
```

Comment:
Do we need this many similar entrypoints (here vs. mc_from_config vs. monte_carlo.py itself)?  Could you consolidate them?

Reply:
Makes sense to take out the one in monte_carlo.py because the methods from there are meant to be imported, so I'll take out the main() function in that.